implementations/contracts/ERC725X.sol

```diff
@@ -21,6 +22,14 @@ contract ERC725X is ERC725XCore {
         if (_newOwner != owner()) {
             OwnableUnset.initOwner(_newOwner);
         }
-        _registerInterface(_INTERFACEID_ERC725X);
+    }
+
+    /* Overrides functions */
+
+    /**
+     * @dev See {IERC165-supportsInterface}.
+     */
+    function supportsInterface(bytes4 interfaceId) public view virtual override returns (bool) {
```

Comment:
It would happen in other contracts (ex: lsp-smart-contracts) but aside from this, I think ERC165 IDs should not be related to the Core logic of the contracts.

Reply:
But this would make any contract that inherits from the `ERC725XCore` or `ERC725YCore` have to implement the `supportsInterface(...)` function. 
To ensure it returns `true` for other contracts that call them with `supportsInterface(erc725X_interface_id)` or supportsInterface(erc725Y_interface_id).

What was the error in the `lsp-smart-contracts` with the inheritance?

Reply:
added the topic to discussion in sc meeting notion

Reply:
ok for where ERC165 is mentioned, as we don't want the Core contracts to focus on the interface detection, but rather on the logic of the interface they implement.